### PR TITLE
Made a parsable players list.

### PR DESCRIPTION
**Pull request by jimmyhmiller**
Players ought to be in a parseable format for easy botting. Yaml is really light weight so I figured it was a good choice.

This is not a rule proposal.

### CODE REVIEW

File: players.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+activePlayers:
+  - name:  ArchmageInc
+    points: 10
+  - name: jimmyhmiller
+    points: 0
+  - name: dilbert88 
+    points: 0
+inActivePlayers: []
```
Comment (dilbertKocik) [inline players.yaml]: FYI, camelCase makes inActivePlayers a bit confusing to read. I'm not saying you should change it. I'm just making an observation.
Edit: but if you did change it, maybe go with inactivePlayers.

Comment (jimmyhmiller) [inline players.yaml]: Yes it should be inactivePlayers. Will fix.

**Comment by ArchmageInc** [comment]
yay

File: players.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+activePlayers:
+  - name:  ArchmageInc
+    points: 10
```
Comment (ArchmageInc) [inline players.yaml]: Also, this should now be 23

**Pull request merged**